partners/project-09.md

```diff
@@ -0,0 +1,61 @@
+---
+altLangPage: "https://conception.canada.ca/partenaires/project-09.html"
+date: 2024-01-02
+dateModified: 2024-01-02
+description: "Partner project: Make employment insurance maternity and parental benefits easier to find and understand for Canadians planning to expand their family. Date: 2019"
+title: "EI maternity and parental benefits"
+---
+<p>Make it easier for Canadian parents to understand and apply for maternity and parental benefits.</p>
+<h2>Date</h2>
+<p>2019</p>
+<h2>Partner departments</h2>
+<ul>
+  <li>Service Canada</li>
+  <li>Employment and Social Development Canada (ESDC)</li>
+</ul>
+<h2>Why this work matters</h2>
+<ul>
+  <li>Thousands of Canadians parents apply for maternity and parental benefits each year</li>
```

Comment:
Please remove 's' from 'Canadian'. So "Thousands of Canadian parents..."

Reply:
Fixed.